DataGenerator/src/server.js

```diff
@@ -1,7 +1,8 @@
 import DATA_FORMAT from "../../Backend/Data/sc1-data-format/format.json";
 import { DateTime } from "luxon";
+import IMPORTED_CONSTANTS from "./constants.json";
```

Comment:
No need to put imported in the name.
```suggestion
import CONSTANTS from "../src/constants.json";
```